### PR TITLE
Offerings section

### PR DESCRIPTION
**Pull request by rlyne**
- Added .gitignore
- Added Offerings section
- Added SPC skeleton
- Added SPC images

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/c66b1165219934c44840d0da8cdf6dd9f1307fcf/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b15fe905ff685b8e0dc42a3a3a2cc9f77514ea35/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/98f30c8b10b94ef5aff23b2f9a345c4c1612b57e/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/57f61ecf568245b41b6375e39d9dba3310c5d3ea/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b2da62c1365a658a5c2f08bbebb085c7bd05598b/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/221c3dea4469d9e682279318b9dc2c286d5f4bce/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/ad9d4cac846bd5b94d71f1f23445e7a661de8300/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/7022855ef89379f6ddb18c5d6c6059894f8af046/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/33aae27b972985681f3a1e87d82ad58b38ee7dcd/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/cf55fbde9056d293f83e84967bf750e159703a8c/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b923a41f993fa9213eed4345e5bef77572a0c2c7/index.html)

File: docs/en/offerings/spc/setup/tuning.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+lang: en
+---
+
+# Fine Tuning
+
+## Cost Management Recommendations
+
+### S3 Infrequent Access
+
+We recommend enabling S3 Infrequent access to reduce easily reduce costs without impacting performance. For the Shotgun Cloud hosted offering, we apply a policy for all objects older than a month.
```
Comment (tannazie) [inline docs/en/offerings/spc/setup/tuning.md]: typo: "reduce easily reduce"

**Pull request merged**